.gitignore

```diff
@@ -1,6 +1,7 @@
 node_modules
 npm-debug.log
 .DS_Store
+.out
```

Comment:
You may also need to add this directory to [_.jpmignore_](https://github.com/mozilla/activity-stream/blob/master/.jpmignore) so it doesn't get bundled into the add-on.